src/utils.js

```diff
@@ -1,7 +1,12 @@
 
 
 export const dateToString = (date) => {
-    return ("0" + date.getDate()).slice(-2) + '.' + ("0" + (date.getMonth()+1)).slice(-2);
+   return ("0" + date.getDate()).slice(-2) + '.' + ("0" + (date.getMonth()+1)).slice(-2);
+};
+
+export const fullDateToString = (date) => {
+   return ("0" + date.getDate()).slice(-2) + '.' + ("0" + (date.getMonth()+1)).slice(-2) + '.' + date.getFullYear();
```

Comment:
```suggestion
   return dateToStrong(date) + '.' + date.getFullYear();
```
Maybe let's reuse the function?

Reply:
Fair, but not dateToStrong